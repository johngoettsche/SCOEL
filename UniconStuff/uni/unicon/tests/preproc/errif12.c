/* test error message: no expression on #elif */

#if 0
one
#elif
two
#endif
