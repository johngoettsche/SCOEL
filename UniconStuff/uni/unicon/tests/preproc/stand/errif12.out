pp: File errif12.c; Line 5: syntax error in condition of #elif
